packages/core/src/modules/clipboard.ts

```diff
@@ -12,12 +12,19 @@ export default class clipboard {
         ele.style.left = "-10000px";
         document.querySelector(".fortune-container")?.append(ele);
       }
+      const previouslyFocusedElement = document.activeElement as HTMLElement;
       ele.style.display = "block";
       ele.innerHTML = str;
       ele.focus({ preventScroll: true });
       document.execCommand("selectAll");
       document.execCommand("copy");
       setTimeout(() => ele?.blur(), 10);
+      if (
+        previouslyFocusedElement &&
+        typeof previouslyFocusedElement.focus === "function"
+      ) {
+        previouslyFocusedElement.focus();
+      }
```

Comment:
```suggestion
      setTimeout(() => {
        ele?.blur();
        previouslyFocusedElement?.focus?.();
      }, 10);
```

I think this is enough.

The luckysheet guys put `blur` inside setTimeout saying "to wait for keyPress event", which doesn't make sense to me. But let's be safe here since it might have something to do with browser compatibility.

Reply:
This makes sense, have tested this change too. Added the commit. Thanks for pointing out.